cmd/opentelemetry/app/defaultcomponents/defaults.go

```diff
@@ -92,25 +92,21 @@ func Components(v *viper.Viper) config.Factories {
 	factories.Exporters[badgerExp.Type()] = badgerExp
 	factories.Receivers[kafkaRec.Type()] = kafkaRec
 
-	jaegerRec := factories.Receivers["jaeger"].(*otelJaegerReceiver.Factory)
 	factories.Receivers["jaeger"] = &jaegerreceiver.Factory{
-		Wrapped: jaegerRec,
+		Wrapped: otelJaegerReceiver.NewFactory(),
```

Comment:
Previously the same factory was passed into both jaegerreceiver.Factory shims.  I don't see any harm in doing it this way, and actually prefer this stylistically.  Unsure if there are consequences I'm unaware of.

Reply:
They removed factory types and instead they use just factory method from the helper package.